addons/spreadsheet/static/src/public_readonly_app/public_readonly.scss

```diff
@@ -1,12 +1,69 @@
+$filter-panel-width: 300px;
+
 #spreadsheet-mount-anchor {
   background-color: #DDDDDD;
-  flex-grow : 1;
+  overflow: hidden;
+  display: flex;
+  flex-direction: row-reverse;
 }
 .o-spreadsheet {
   font-size: 12px;
   height: 100%;
+  flex-grow: 1;
 
   .o-topbar-toolbar .o-readonly-toolbar.text-muted {
       color: $info !important;
   }
 }
+
+.o-public-spreadsheet-filters{
```

Comment:
the overall design could be improved, let's wait for François testing if he has design suggestions/ideas